rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -326,41 +396,59 @@ extern "C"  RocalTensor  ROCAL_API_CALL rocalJitter(RocalContext context, RocalT
 /// \param max
 /// \param is_output
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalJitterFixed(RocalContext context, RocalTensor input,
-                                                        int kernel_size, bool is_output);
+extern "C" RocalTensor ROCAL_API_CALL rocalJitterFixed(RocalContext context, RocalTensor input,
+                                                       int kernel_size, bool is_output, int seed = 0,
+                                                       RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
```

Comment:
Not sure what is the better way to do it
In some cases we are adding new parameter in new line, in some cases we are combining multiple params in single line. Check the difference better rocalJitter, rocalJitterFixed. If applicable, let us follow one convention

I understand when the type name itself is too long, the line will look very long, but for this case we can combine in single line i guess